Assets/Scripts/Entity/Player.cs

```diff
@@ -10,6 +10,9 @@ class Player : AbstractEntity
 
         public int collectible;
         public Text counter;
+        public float enemyBumpPowerY = 10f;
+        public float enemyBumpPowerX = 10f;
+        public int health = 3;
```

Comment:
have a field for max health as well, since the player needs to be sent back to the spawn position when he dies.

Reply:
Added max health field and player gets reset to max health on death. Also player gets transported to spawn as before.